debug_exception.py

```
@@ -0,0 +1,64 @@
+from mad_spark_alt.core import SmartQADIOrchestrator
+
+async def test():
+    orchestrator = SmartQADIOrchestrator()
+    
+    try:
+        await orchestrator.ensure_agents_ready()
+        print("✅ Agents ready")
+        
+        result = await orchestrator.run_qadi_cycle(
+            problem_statement="Test",
+            cycle_config={"max_ideas_per_method": 1}
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)